### ISSUE TITLE
Build failed. Compilation error

### ISSUE THREAD

**Issue by wanderodrigues** [FlutterFlow/flutterflow-issues#3244]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I checked the error on June 25, 2024. Until the last time I accessed it on June 21, 2024, it was working. When running the Test Mode, it returns a compilation error as described in the field below.

### Expected Behavior

that only the compiler worked.

### Steps to Reproduce

To simulate, just try running the Test Mode to compile the application.
![Error](https://github.com/FlutterFlow/flutterflow-issues/assets/155364900/b2f17f96-5dd3-46ae-809f-de188afc8454)

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT88lfLqy49IsctD+Krqde9ajikvJjs/a+YwluxqEAg1F73qB4l3Z8/7VEpAXsThYGxteU22gVgBpqHKuPPhEsECBwSXGKY5/6liTjnzcHKzaq6OC5bhVXckEZlbfRXAy7er2SMmPs1fW14Sw0itLOiidDHAQ5DpCzJNDv2JEKLAp2/nABzdFyZE828URVOgCfUUTxGByM+UtpJh0eDnyw==

### Visual documentation

56 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
Launching lib/main.dart on Web Server in debug mode...
Waiting for connection from debug service on Web Server...      
Warning: In index.html:51: Local variable for "serviceWorkerVersion" is deprecated. Use "{{flutter_service_worker_version}}" template token instead.
Warning: In index.html:74: "FlutterLoader.loadEntrypoint" is deprecated. Use "FlutterLoader.load" instead.
lib/flutter_flow/flutter_flow_charts.dart:35:15: Error: No named parameter with the name 'getTooltipColor'.
              getTooltipColor: (group) =>
              ^^^^^^^^^^^^^^^
/root/.pub-cache/hosted/pub.dev/fl_chart-0.62.0/lib/src/chart/line_chart/line_chart_data.dart:1290:3: Context: Found this candidate, but the arguments don't match.
  LineTouchTooltipData({
  ^^^^^^^^^^^^^^^^^^^^
lib/flutter_flow/flutter_flow_charts.dart:157:13: Error: No named parameter with the name 'getTooltipColor'.
            getTooltipColor: (group) =>
            ^^^^^^^^^^^^^^^
/root/.pub-cache/hosted/pub.dev/fl_chart-0.62.0/lib/src/chart/bar_chart/bar_chart_data.dart:698:3: Context: Found this candidate, but the arguments don't match.
  BarTouchTooltipData({
  ^^^^^^^^^^^^^^^^^^^
/root/.pub-cache/hosted/pub.dev/fl_chart-0.62.0/lib/src/utils/utils.dart:269:20: Error: Member not found: 'MediaQuery.boldTextOverride'.
    if (MediaQuery.boldTextOverride(context)) {
                   ^^^^^^^^^^^^^^^^
/root/.pub-cache/hosted/pub.dev/infinite_scroll_pagination-3.2.0/lib/src/ui/default_indicators/first_page_exception_indicator.dart:27:50: Error: The getter 'headline6' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/sdks/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'headline6'.
              style: Theme.of(context).textTheme.headline6,
                                                 ^^^^^^^^^
Waiting for connection from debug service on Web Server...         52.6s
Failed to compile application.

### Environment

```markdown
- FlutterFlow version: 4.1.63+
- Platform: Mobile
- Browser name and version: Chrome Version126.0.6478.116 (Official compilation.) (64 bits)
- Operating system and version affected: Windows 10
```

**Comment by rzambroni** [comment]
Hey @wanderodrigues, yesterday's update included an upgrade to the 3.22.2 version of Flutter, you might need to check your custom code for dependencies issues and fix them. 

In case it is helpful here is a quick guide on how to fix dependency issues: [https://www.youtube.com/watch?v=8Y1uyCC_dXE](https://www.youtube.com/watch?v=8Y1uyCC_dXE)

Since the issues are related i'm closing  issue in favor of: [https://github.com/FlutterFlow/flutterflow-issues/issues/3229](https://github.com/FlutterFlow/flutterflow-issues/issues/3229)

**Comment by wanderodrigues** [comment]
We solved it by updating two libraries: infinite_scroll_pagination: ^4.0.0 and fl_chart: ^0.68.0.
